connect/src/com/telenor/connect/ui/ConnectActivity.java

```diff
@@ -17,12 +19,17 @@
 import com.telenor.connect.utils.ConnectUrlHelper;
 import com.telenor.connect.utils.ConnectUtils;
 
+import java.util.HashSet;
+import java.util.Iterator;
```

Comment:
I can't immeadiately tell where the Iterator and List imports are being used. Remove in case they're not used in this class.

Reply:
Ah, thanks. Refactored the implementation, but forgot to remove these.

Reply:
Resolved by ae4ca8b